xonsh/commands_cache.py

```diff
@@ -50,10 +50,11 @@ def get_possible_names(name):
          name on Windows as a list, conserving the ordering in `PATHEXT`.
          Returns a list as `name` being the only item in it on other platforms."""
         if ON_WINDOWS:
+            pathext = builtins.__xonsh_env__.get('PATHEXT', ['.EXE','.BAT','.CMD'])
```

Comment:
Well it is needed if  `__xonsh_env__` is mocked as a simple empty dict. But I guess it is better to just add needed data where we mock `__xonsh_env__`